DESCRIPTION

```diff
@@ -23,6 +23,7 @@ Authors@R: c(
     person("Rana", "Gahwagy", role="ctb", email = "r.w.g1@hotmail.com"),
     person("Natalia", "Iannucci", email = "iannuccinatalia@gmail.com", role = "ctb"),
     person("Marium", "Tapal", email = "mariumtapal@gmail.com", role = "ctb"),
+    person("Lizette", "Carpenter", email = "lizettecarpenter@gmail.com", role = "ctb"))
```

Comment:
Hey @lcarpenter20, thanks for your PR. I can't test your package b/c the code you've submitted has a mismatched comma and parentheses here. Did you make sure to test that your package actually builds before submitting your code?Before you make any commits, it's important for you to always:
1. Test if the package will build
1. Run the package checks

After you've fixed this, I can test out your changes. Tagging @beanumber